### PR TITLE
Ping: Add two relevant buttons

### LINKED ISSUE #210: Ping: Add relevant buttons
Two buttons can be added:
* Is it up? link e.g. "https://isitup.org/google.com"
* Go to website link e.g. "google.com"
Feel free to suggest more!

### PR DESCRIPTION
**Pull request by willmichael**
Resolves #210, couple lines added to create two button links in the ping module. This would be my first contribution, albeit small, to an OS project so any advice on PR/Commit messages, code/comments, or testing would be appreciated, thanks!

### CODE REVIEW

File: modules/src/ping.py [hunk 1]
```diff
@@ -27,9 +27,15 @@ def process(input, entities):
             image_url = 'http://fa2png.io/media/icons/font-awesome/4-6-3/exclamation-circle/256/0/f1c40f_none.png'
         else:
             raise Exception("Something unexpected happened!")
+        
+        is_up_url = 'https://isitup.org/' + hostname
+        buttons = ButtonTemplate()
+        buttons.add_web_url('isitup.org Link' , is_up_url)
```
Comment (swapagarwal) [inline modules/src/ping.py]: Name it `View more info`

File: modules/src/ping.py [hunk 2]
```diff
@@ -27,9 +27,15 @@ def process(input, entities):
...
+        buttons.add_web_url('Site Link', url)
```
Comment (swapagarwal) [inline modules/src/ping.py]: Point to `hostname` here. Name it `Go to website`

**Comment by swapagarwal** [comment]
@willmichael Welcome to the awesome world of open-source! The PR looks good functionality-wise. Address the few comments above, and this should be good to go! 😄

**Comment by willmichael** [comment]
I went ahead and implemented the changes you requested, let me know if anything else needs work

**Comment by swapagarwal** [comment]
Thanks @willmichael! 👍

**Pull request merged**